canonical/ubuntu-pro-client
# help: remove service list from help output

## Why is this needed?

This PR solves all of our problems because we don't call the backends anymore when just running `help`.

OOOOOOH but we are missing the service list now
yes
that's the point
it is not there in the new UX design, and getting rid of it now prevents problems

Fixes: #3126

## Test Steps

Disconnect your system
Run `pro help` or `pro -h` or `pro --help` and see it works and has less content
Integration tests are smaller too

---
